netlify.toml

```diff
@@ -4,3 +4,9 @@ package="netlify-plugin-checklinks"
 [build]
   publish = "_site/"
   command = "yarn build"
+
+[[plugins]]
+    package = "netlify-plugin-11ty"
+    [plugins.inputs]
+        cache_img = './assets/images'
+        cache_img_httpHeader = false
```

Comment:
Does setting `cache_img_httpHeader` to `false` disable caching for images, or is this for something else?